modules/dbparser/tests/test_parsers_e2e.c

```diff
@@ -38,18 +38,22 @@ do { \
 
 
 gchar *pdb_parser_skeleton_prefix ="<?xml version='1.0' encoding='UTF-8'?>\
-          <patterndb version='3' pub_date='2010-02-22'>\
-            <ruleset name='test1_program' id='480de478-d4a6-4a7f-bea4-0c0245d361e1'>\
-                <pattern>test</pattern>\
-                    <rule id='1' class='test1' provider='my'>\
-                        <patterns>\
-                         <pattern>";
-
-gchar *pdb_parser_skeleton_postfix =  "</pattern>\
-                      </patterns>\
-                    </rule>\
-            </ruleset>\
-        </patterndb>";
+<patterndb version='4' pub_date='2010-02-22'>\
+  <ruleset name='test1_program' id='480de478-d4a6-4a7f-bea4-0c0245d361e1'>\
+    <patterns>\
+      <pattern>test</pattern>\
+    </patterns>\
+    <rules>\
+      <rule id='1' class='test1' provider='my'>\
+        <patterns>\
+          <pattern>" /* HERE COMES THE GENERATED PATTERN */ ;
+
+gchar *pdb_parser_skeleton_postfix =  /* HERE IS THE GENERATED PATTERN */ "</pattern>\
```

Comment:
I like the terminology `suffix` better. Could you perhaps add some modifiers as well, like `static const`?